### PR TITLE
docs: add changelog 4.18.4

### PR DESCRIPTION
**Pull request by vagusX**
- Typography
  - ⚡️ 优化 Typography 在配置 `tooltip` 时优先使用原生省略样式以提升性能。[#33669](https://github.com/ant-design/ant-design/pull/33669)
  - 🐞 重构 Typography `ellipsis` 逻辑以修复 `children` 如果消费上游 Context 会报错的问题。 [#33725](https://github.com/ant-design/ant-design/pull/33725)
- Icon
  - 🐞 修复 `<Icon component={HomeOutlined} />`; 和 `<HomeOutlined />`; 不对齐的问题。[#33709](https://github.com/ant-design/ant-design/pull/33709)
  - 🐞 修复 `<Icon component={SyncOutlined} spin />`; 抖动的问题。[#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
- Input
  - 🐞 修复 Input 相关组件设置 `hidden` 时的展示问题。[#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge)，[#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
  - 🐞 修复设置 `prefix`/`suffix` 的 Input 的 `hidden` 无效的问题。
  - 🐞 修复 Input 传入 `showCount` 时控制台 warning 提示问题。[#33686](https://github.com/ant-design/ant-design/pull/33686) [@whwangms](https://github.com/whwangms)
- ⚡️ 修复 ConfigProvider 和 Anchor 的渲染函数多次运行的问题。[#33723](https://github.com/ant-design/ant-design/pull/33723) [@mrwd2009](https://github.com/mrwd2009)
- 🐞 修复 Cascader 组件中出现重复 key 的 warning 问题。[#33649](https://github.com/ant-design/ant-design/pull/33649) [@dimbslmh](https://github.com/dimbslmh)
- 🐞 Checkbox.Group 的 `options` 支持数组中直接传入 number 和 boolean 类型。[#33678](https://github.com/ant-design/ant-design/pull/33678)
- 🐞 修复 Form `validateMessages` 在多个 ConfigProvider 内错乱的问题。[#33705](https://github.com/ant-design/ant-design/pull/33705)
- 🐞 修复 Steps 组件在 `type` 为 navigation 和 `labelPlacement` 为 vertical 时，tail 部分不会显示的问题。[#33716](https://github.com/ant-design/ant-design/pull/33716) [@toSayNothing](https://github.com/toSayNothing)
- 🐞 修复 Image 底部留白问题。[#33631](https://github.com/ant-design/ant-design/pull/33631) [@fanerge](https://github.com/fanerge)
- 🇰🇭 修正高棉语 (km_KH) 语言包中部分翻译。[#33738](https://github.com/ant-design/ant-design/pull/33738) [@vireakkeosokvibol](https://github.com/vireakkeosokvibol)

----

- Typography
  - ⚡️ Optimize Typography to use native css ellipsis when `tooltip` is configured. [#33669](https://github.com/ant-design/ant-design/pull/33669)
  - 🐞 Refactor Typography `ellipsis` logic to fix error when `children` use Context content. [#33725](https://github.com/ant-design/ant-design/pull/33725)
- Icon
  - 🐞 Fix the alignment for `<Icon component={HomeOutlined} />` and `<HomeOutlined />`. [#33709](https://github.com/ant-design/ant-design/pull/33709)
  - 🐞 Fix the shaking for `<Icon component={SyncOutlined} spin />`. [#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
- Input
  - 🐞 Fix the display effect for Input related components with `hidden`. [#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge), [#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
  - 🐞 Fix the warning console prompt for Input with `showCount`. [#33686](https://github.com/ant-design/ant-design/pull/33686) [@whwangms](https://github.com/whwangms)
- ⚡️ Fix ConfigProvider and Anchor rerender unexpectedly. [#33723](https://github.com/ant-design/ant-design/pull/33723) [@mrwd2009](https://github.com/mrwd2009)
- 🐞 Fix the duplicate keys warning in Cascader. [#33649](https://github.com/ant-design/ant-design/pull/33649) [@dimbslmh](https://github.com/dimbslmh)
- 🐞 Checkbox.Group support number and boolean as options element. [#33678](https://github.com/ant-design/ant-design/pull/33678)
- 🐞 Fix Form `validateMessages` bug in multiple ConfigProvider. [#33705](https://github.com/ant-design/ant-design/pull/33705)
- 🐞 Fix the tail part would be hidden when `type` is navigation and `labelPlacement` is vertical in Steps. [#33716](https://github.com/ant-design/ant-design/pull/33716) [@toSayNothing](https://github.com/toSayNothing)
- 🐞 Fix the bottom blank for Image. [#33631](https://github.com/ant-design/ant-design/pull/33631) [@fanerge](https://github.com/fanerge)
- 🇰🇭 Fix some translations for Khmer (km_KH). [#33738](https://github.com/ant-design/ant-design/pull/33738) [@vireakkeosokvibol](https://github.com/vireakkeosokvibol)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
 
+## 4.18.4
+
+`2022-01-18`
+
+- Typography
+  - ⚡️ Optimize Typography to use native css ellipsis when `tooltip` is configured. [#33669](https://github.com/ant-design/ant-design/pull/33669)
+  - 🐞 Refactor Typography `ellipsis` logic to fix error when `children` use Context content. [#33725](https://github.com/ant-design/ant-design/pull/33725)
+- Icon
+  - 🐞 Fix the alignment for &lt;Icon component={HomeOutlined} /&gt; and &lt;HomeOutlined /&gt;. [#33709](https://github.com/ant-design/ant-design/pull/33709)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:26]: `&lt;` => `<Icon component={HomeOutlined} />`

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
 
+## 4.18.4
+
+`2022-01-18`
+
+- Typography
+  - ⚡️ 优化 Typography 在配置 `tooltip` 时优先使用原生省略样式以提升性能。[#33669](https://github.com/ant-design/ant-design/pull/33669)
+  - 🐞 重构 Typography `ellipsis` 逻辑以修复 `children` 如果消费上游 Context 会报错的问题。 [#33725](https://github.com/ant-design/ant-design/pull/33725)
+- Icon
+  - 🐞 修复 &lt;Icon component={HomeOutlined} /&gt; 和 &lt;HomeOutlined /&gt; 不对齐的问题。[#33709](https://github.com/ant-design/ant-design/pull/33709)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:26]:
```suggestion
  - 🐞 修复 `<Icon component={HomeOutlined} />` 和 `<HomeOutlined />` 不对齐的问题。[#33709](https://github.com/ant-design/ant-design/pull/33709)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+  - 🐞 修复 &lt;Icon component={SyncOutlined} spin /&gt; 抖动的问题。[#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
+- Input
+  - 🐞 修复 Input 相关组件设置 `hidden` 时的展示问题。[#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge)
+  - 🐞 修复设置 `prefix`/`suffix` 的 Input 的 `hidden` 无效的问题。[#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:30]: 上面这俩可以合并写，同一个问题，不同的解决范围。

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+  - 🐞 修复 Input 传入 `showCount` 时控制台 warning 提示问题。[#33686](https://github.com/ant-design/ant-design/pull/33686) [@whwangms](https://github.com/whwangms)
+- 🐞 修复 Cascader 组件中出现重复 key 的 warning 问题。[#33649](https://github.com/ant-design/ant-design/pull/33649) [@dimbslmh](https://github.com/dimbslmh)
+- 🐞 Checkbox.Group 的 `options` 支持数组中直接传入 number 和 boolean 类型。[#33678](https://github.com/ant-design/ant-design/pull/33678)
+- 🐞 修复 Form `validateMessages` 在多个 ConfigProvider 内错乱的问题。[#33705](https://github.com/ant-design/ant-design/pull/33705)
+- 🐞 修复 ConfigProvider 和 Anchor 的渲染函数多次运行的问题。[#33723](https://github.com/ant-design/ant-design/pull/33723) [@mrwd2009](https://github.com/mrwd2009)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:35]: 可以用性能的 emoji

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -26,8 +26,7 @@ timeline: true
   - 🐞 Fix the alignment for `<Icon component={HomeOutlined} />` and `<HomeOutlined />`. [#33709](https://github.com/ant-design/ant-design/pull/33709)
   - 🐞 Fix the shaking for `<Icon component={SyncOutlined} spin />`. [#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
 - Input
-  - 🐞 Fix the display problem for Input related components with `hidden`. [#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge)
-  - 🐞 Fix the problem that `hidden` is invalid for Input with `prefix`/`suffix`. [#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
+  - 🐞 Fix the display effect for Input related components with `hidden`. [#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge), [#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
```
Comment (afc163) [inline CHANGELOG.en-US.md:29]:
```suggestion
  - 🐞 Fix the display effect for Input related components with `hidden`. [#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge) [#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
```

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -26,8 +26,8 @@ timeline: true
   - 🐞 修复 `<Icon component={HomeOutlined} />`; 和 `<HomeOutlined />`; 不对齐的问题。[#33709](https://github.com/ant-design/ant-design/pull/33709)
   - 🐞 修复 `<Icon component={SyncOutlined} spin />`; 抖动的问题。[#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
 - Input
-  - 🐞 修复 Input 相关组件设置 `hidden` 时的展示问题。[#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge)
-  - 🐞 修复设置 `prefix`/`suffix` 的 Input 的 `hidden` 无效的问题。[#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
+  - 🐞 修复 Input 相关组件设置 `hidden` 时的展示问题。[#33735](https://github.com/ant-design/ant-design/pull/33735) [@fanerge](https://github.com/fanerge)，[#33706](https://github.com/ant-design/ant-design/pull/33706) [@hydraZty](https://github.com/hydraZty)
+  - 🐞 修复设置 `prefix`/`suffix` 的 Input 的 `hidden` 无效的问题。
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:30]: 这个还在。

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
 
+## 4.18.4
+
+`2022-01-18`
+
+- Typography
+  - ⚡️ Optimize Typography to use native css ellipsis when `tooltip` is configured. [#33669](https://github.com/ant-design/ant-design/pull/33669)
+  - 🐞 Refactor Typography `ellipsis` logic to fix error when `children` use Context content. [#33725](https://github.com/ant-design/ant-design/pull/33725)
+- Icon
+  - 🐞 Fix the alignment for `<Icon component={HomeOutlined} />` and `<HomeOutlined />`. [#33709](https://github.com/ant-design/ant-design/pull/33709)
```
Comment (afc163) [inline CHANGELOG.en-US.md:26]: https://github.com/ant-design/ant-design/pull/33709 里已经提供了英文 changelog。

其他同理，可以都看看。

Comment (vagusX) [reply CHANGELOG.en-US.md:26]: 我觉得那个 PR 的英文描述不够精准，其他 PR 里面也是一样，都用 google translate 了下...

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
 
+## 4.18.4
+
+`2022-01-18`
+
+- Typography
+  - ⚡️ 优化 Typography 在配置 `tooltip` 时优先使用原生省略样式以提升性能。[#33669](https://github.com/ant-design/ant-design/pull/33669)
+  - 🐞 重构 Typography `ellipsis` 逻辑以修复 `children` 如果消费上游 Context 会报错的问题。 [#33725](https://github.com/ant-design/ant-design/pull/33725)
+- Icon
+  - 🐞 修复 `<Icon component={HomeOutlined} />`; 和 `<HomeOutlined />`; 不对齐的问题。[#33709](https://github.com/ant-design/ant-design/pull/33709)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:26]:
```suggestion
  - 🐞 修复 `<Icon component={HomeOutlined} />` 和 `<HomeOutlined />` 不对齐的问题。[#33709](https://github.com/ant-design/ant-design/pull/33709)
```

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+  - 🐞 修复 `<Icon component={SyncOutlined} spin />`; 抖动的问题。[#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:27]:
```suggestion
  - 🐞 修复 `<Icon component={SyncOutlined} spin />` 抖动的问题。[#33726](https://github.com/ant-design/ant-design/pull/33726) [@JX-Zhuang](https://github.com/JX-Zhuang)
```

File: CHANGELOG.en-US.md [hunk 10]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+  - 🐞 Fix the warning in console for Input with `showCount`. [#33686](https://github.com/ant-design/ant-design/pull/33686) [@whwangms](https://github.com/whwangms)
+- ⚡️ Fix ConfigProvider and Anchor rerender unexpectedly. [#33723](https://github.com/ant-design/ant-design/pull/33723) [@mrwd2009](https://github.com/mrwd2009)
+- 🐞 Fix the duplicate keys warning in console for Cascader. [#33649](https://github.com/ant-design/ant-design/pull/33649) [@dimbslmh](https://github.com/dimbslmh)
+- 🐞 Checkbox.Group support number and boolean as options element. [#33678](https://github.com/ant-design/ant-design/pull/33678)
+- 🐞 Fix Form `validateMessages` bug in multiple ConfigProvider. [#33705](https://github.com/ant-design/ant-design/pull/33705)
+- 🐞 Fix the tail part would be hidden when `type` is navigation and `labelPlacement` is vertical in Steps. [#33716](https://github.com/ant-design/ant-design/pull/33716) [@toSayNothing](https://github.com/toSayNothing)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:35]: >  Fix Steps the tail part would be hidden when `type` is navigation and `labelPlacement` is vertical.

File: CHANGELOG.en-US.md [hunk 11]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 Fix the bottom blank for Image. [#33631](https://github.com/ant-design/ant-design/pull/33631) [@fanerge](https://github.com/fanerge)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:36]: > Fix Image has bottom blank style.

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/33755 要补一下 changelog

**Pull request merged**